### PR TITLE
nameday module added

### PR DESCRIPTION
**Pull request by krinistof**
I've created a module for asking for todays nameday!

### CODE REVIEW

File: modules/name_day.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from requests import get
+import json
+
+def todays_names(line):
```
Comment (svimanet) [inline modules/name_day.py]: The function requires a paramter 'line' which is never used. The paramter is also not specified in the actuator call (line 124 of main.py), which causes a TypeError when called. If you remove the unused paramter I would be more than happy to merge the PR :)

``` suggestion
def todays_names():
```

Comment (krinistof) [reply modules/name_day.py]: Yeah, right. I messed it up.

Comment (krinistof) [reply modules/name_day.py]: Hotfix have been pushed.

**Review by svimanet — APPROVED** [review]

**Comment by krinistof** [comment]
Does making feautures to it be counted as PR for hacktoberfest?
I wanted to pass the line for parameters of language, or even date

**Comment by svimanet** [comment]
Yes, Pull Requests to this repository should count towards your hacktoberfest contributions, since it's a licensed open-source project with all community guidelines in place. You can check your Pull Requests on the hacktoberfest site: https://hacktoberfest.digitalocean.com/profile
Each PR has a review period where the maintainers of the repository can reject or approve the PR. This PR has already been approved and merged, and should become valid when the review period of 7 days ends.

If you want to make more Pull Requests to this repository, you are more than welcome to!

**Pull request merged**